deepchem/feat/roberta_tokenizer.py

```diff
@@ -57,8 +58,14 @@ def _featurize(self, datapoint: str, **kwargs) -> List[List[int]]:
         """
 
         # the encoding is natively a dictionary with keys 'input_ids' and 'attention_mask'
-        encoding = list(self(datapoint, **kwargs).values())
+        encoding = list(super().__call__(datapoint, **kwargs).values())
         return encoding
 
-    def __call__(self, *args, **kwargs) -> Dict[str, List[int]]:
-        return super().__call__(*args, **kwargs)
+    def __call__(self,
+                 datapoints,
+                 padding: bool = True,
+                 **kwargs) -> Dict[str, List[int]]:
```

Comment:
Given we are not directly calling the superclass, let's add a brief docstring here

Reply:
Please see the latest commit